src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceField.cs

```diff
@@ -43,7 +43,7 @@ internal partial class CSharpIntroduceVariableService
                         SyntaxFactory.VariableDeclarator(
                             newNameToken.WithAdditionalAnnotations(RenameAnnotation.Create()),
                             null,
-                            SyntaxFactory.EqualsValueClause(expression))))).WithAdditionalAnnotations(Formatter.Annotation);
+                            SyntaxFactory.EqualsValueClause(expression.WithoutTrailingTrivia().WithoutLeadingTrivia()))))).WithAdditionalAnnotations(Formatter.Annotation);
```

Comment:
:bulb: There is a `WithoutTrivia()` extension method which behaves like these two combined.

Reply:
Done. I also applied this simplification in csharpintroducevariableservice_introducelocal.cs